src/System.Net.NetworkInformation/src/project.json

```diff
@@ -2,7 +2,9 @@
   "dependencies": {
     "System.Diagnostics.Contracts": "4.0.0-*",
     "System.Threading": "4.0.0-*",
-    "System.Net.Primitives": "4.0.10-beta-*"
+    "System.Net.Primitives": "4.0.10-beta-*",
+    "System.Net.Sockets": "4.0.10-beta-*",
+    "System.Net.Utilities": "4.0.0-beta-*"
```

Comment:
Is this dependency accurate? I didn't think any internal code depended on S.N.Utilities.